core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -688,6 +675,33 @@ class TestCaseWriter {
         }
     }
 
+    private fun handleAdditionalFieldValues(stringKey: String, resContentsItem: Any?): List<Pair<String, String>>?{
```

Comment:
please add some JavaDocs, especially about what are the pairs of strings returned